Docs: Initial advanced config section

nickchase opened the pull request:
Initial advanced config section.  The custom plugins section is written.  NIC bonding is there, but just the raw material.  The quantum HA document is there but just as a stub.  Also corrected IP addresses in instructions.

Commit:
Workaround to PKI token issues -> use UUID as defaults.

Because of unresolved "Error 400 on SERVER: undefined method `<<' for {}:Hash" issue,
set 'token_format = UUID' to preserve default behavior from <= Folsom as a workaround

Fixes: defect #FUEL-538

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Add "shared" parameter quantum net classes, change net04_ext to be created as shared

Fixes: defect #FUEL-546

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
nova conductor support

Commit:
add file conductor

Commit:
Add params for nova-cond

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Restore deleted compute serv name param

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix ensure_package, enabled for nova-cond

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix 'shared' key for quantum provider

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
disable gpgchek

Commit:
Fix typo for :shared param for quantum net provider

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fixes: defect #FUEL-548

Commit:
Fix prepare.py for simple usecase

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Workaround preapre.py controller IP issue for non HA simple case

Use public IP of 1st controller node,
ignore 2 other controller nodes which are non used,
but deployed by jenkins.

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix spacing typo

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Add catalog_type for identity to tempest sample conf

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Add more identity related values to tempest conf sample

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix prepare.py, Add grizzly_simple, grizzly tempest sample conf.

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix comma

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix brackets

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Add vars for tempest grizzly config

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
fixed horizon bug FUEL-555

Commit:
disable syslog

Commit:
horizon fix permission

Commit:
subscribe add

Commit:
Fix /var/log/keystone owner

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix recurse and require for keystone files

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
fix keystone loop

Commit:
keystone db_sync from keystone user

Commit:
Revert "Fix recurse and require for keystone files"

This reverts commit 6ea2f7a1618102eaba0cd415d9adb91e05d4a61f.

Commit:
Revert "Fix /var/log/keystone owner"

This reverts commit c7797911a6f6b28be085965320e703f1357de970.

Commit:
Fix runas for keystone-manage pki_setup

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix ADMIN_USER_NAME for prepare.py

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix mess with ADMIN_USER_NAME

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix sample config for grizzly tempest

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Add public router & net id to tempest conf for grizzly

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Render public net & router id to tempest conf

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Set tempest config to run quantum tests

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

aglarendil committed:
[FUEL-526] skip openstack::cinder class if it is missing

Conflicts:

	deployment/puppet/openstack/manifests/controller.pp

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

aglarendil committed:
[FUEL-526] skip openstack::cinder class if it is missing

Commit:
Fix bracket typo

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix typo in func call

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix quantum client list casts

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Use quantumclient.v2_0, fix get router_id

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Make prepare.py for tempest reentrable

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix missing colons

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix def order

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix missed 'self's, change get*->_get* names

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix id retrieving for ext net & ext router

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Set cinder on computes for simple example

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
remove 'order' parameter

Commit:
Fix typo

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
ubuntu upstream grizzly repos

Commit:
Revert "Fix write_openstack_simple_manifest and prepare.py Init for simple example"

This reverts commit bc9a2cc488b05a3d1356340d566e9fea255dbc82.

Conflicts:

	fuel_test/manifest.py

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix fixed_network for grizzly tempest config. Undo nv_physical_volume to defaults.

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Fix typo

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
Revert "Fix typo"

This reverts commit 1ceb5f77f5fa689348ac2d7cc5085b9f8ceee6ac.

Commit:
Revert "Fix fixed_network for grizzly tempest config. Undo nv_physical_volume to defaults."

This reverts commit 10a2bf01f8353ffb8475fcf17dc50ab7adb8698f.

Commit:
Change tenant fixeds & volume intervals for tempest conf, undo nv_phys_vol for prep

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
reformat

Commit:
FIX: FUEL-565

Commit:
Add ssh tests, fix network_for_ssh value

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
enable folsom repos

Commit:
remove icorrect apt-pin

Commit:
Add use LBM for LiveMigration to tempest config

Signed-off-by: Bogdan Dobrelya <bdobrelia@mirantis.com>

Commit:
nova conductor correct name

bogdando committed:
Merge remote-tracking branch 'stable/master' into grizzly

Port changes from Mirantis/master

Conflicts:
	deployment/puppet/openstack/examples/site_openstack_single.pp
	deployment/puppet/openstack/manifests/compute.pp
	deployment/puppet/openstack/manifests/controller.pp
	deployment/puppet/openstack/manifests/mirantis_repos.pp

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Restore yum-priority install

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge remote-tracking branch 'zipfer/grizzly' into grizzly

Fix merge errors, add ubuntu support

Conflicts:
	deployment/puppet/keystone/manifests/init.pp
	deployment/puppet/openstack/manifests/compute.pp
	deployment/puppet/openstack/manifests/mirantis_repos.pp
	fuel_test/config/tempest.conf.grizzly.sample
	fuel_test/prepare.py

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Fix spacing

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Add README.rst skeleton for grizzly CI job example

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
CI fix - extend public pull x2 to allow TEMPEST run intances

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Temporary change for public IP (CI centos grizzly)

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Fix typo

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Fix temporary change for public IP (CI centos grizzly)

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Fix quotes typo

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
CI Change prefix for allocation pools for ext net

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Revert "CI fix - extend public pull x2 to allow TEMPEST run intances"

This reverts commit d6aabdac9b5002ce2a93b03b7687c5db87984306.

Conflicts:

	fuel_test/settings.py

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Add CHANGELOG for grizzly centos simple research

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
CI fix network name for ssh

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

Commit:
grizzly osci repo

Commit:
add cloud-archive to depend

bogdando committed:
Change some tempest config keys

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Change some tempest config keys & CI readme

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Change README.rst for CI

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
CI Change private net for SSH

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

Commit:
moved nova-conductor to controller

Commit:
gitignore .idea

bogdando committed:
Revert "Fix temporary change for public IP (CI centos grizzly)"

This reverts commit c777b93cfeae86ebffb2fd0cafcee200133cc530.

Conflicts:

	fuel_test/settings.py

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
CI Fix for tenant network cidr, public pool set default routed

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
CI change for not routable public pool, ssh net = ext

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
CI change README.rst

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
CI README.rst add example for single tempest test run

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Restore Tenants nets routable True

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Revert "CI Fix for tenant network cidr, public pool set default routed"

This reverts commit 79619a2e1e82be1e6db0cf0cc5a1d4a989396d53.

bogdando committed:
Revert "CI fix - extend public pull x2 to allow TEMPEST run intances"

This reverts commit d6aabdac9b5002ce2a93b03b7687c5db87984306.

Conflicts:

	fuel_test/settings.py

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
CI Change tenant net CIDR

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge branch 'master' into grizzly

Conflicts:
	deployment/puppet/openstack/examples/site_openstack_simple.pp
	deployment/puppet/openstack/manifests/compute.pp
	fuel_test/manifest.py

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge branch 'grizzly' of github.com:Zipfer/fuel into Zipfer-grizzly

Conflicts:
	deployment/puppet/openstack/manifests/mirantis_repos.pp

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
CI Change readme

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #83 from bogdando/grizzly

CI Change readme

bogdando committed:
Fix simple:controller.cinder var

Fixes: defect #FUEL-598

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #85 from bogdando/grizzly

Fix simple:controller.cinder var

bogdando committed:
Revert "CI Change tenant net CIDR"

This reverts commit 305a2724156e98e0ccf3dfbcbcb24fa90d94f8dd.

bogdando committed:
Change net04 to shared, CI tenants net unreachable

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Change net04 to be created before net04_ext

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Revert "Change net04 to be created before net04_ext"

This reverts commit f1f17bebda36591b452da1f25617c8f0cfec3fc0.

bogdando committed:
Merge pull request #87 from bogdando/grizzly

Grizzly CI

bogdando committed:
CI Add DEBUG for pupet

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Rsyslog:Disk-Assisted Memory Queues, async writes

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

Commit:
ip changes

Commit:
Add initial "advanced configuration" section.

Add first cut of "advanced configuration".  Only the introduction and
the "plugins" page are written; the rest are placeholders.

Commit:
Custom plugin draft.  Bonding raw material

adanin commented on docs/pages/advanced-topics/0030-quantum-HA.rst:
```diff
@@ -0,0 +1,7 @@
+OpenStack Networking HA
+-----------------------
+
+Fuel 2.2 introduces support for OpenStack Networking (formerly known as Quantum) in a high-availability  configuration. To accomplish this, fuel uses a combination of Pacemaker and Corsync to ensure that if the networking service goes down, it will be restarted,, either on the existing node or on separate node.
```
Misspelling in Corosync.
Comma duplication after "it will be restarted"

plechenko replied:
Double space at "high-availability  configuration"

plechenko replied:
fuel should be written as Fuel

bogdando committed:
Merge remote-tracking branch 'stable/master' 2.2-folsom-79-g13cd2ad into grizzly

Conflicts:
	deployment/puppet/openstack/examples/site_openstack_simple.pp
	deployment/puppet/openstack/manifests/mirantis_repos.pp

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Edit CHANGELOG

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #143 from bogdando/grizzly

Grizzly merge from master 2.2-folsom-79-g13cd2ad

bogdando committed:
Merge pull request #144 from bogdando/ci_debug_puppet

CI Add DEBUG for pupet

bogdando committed:
Set EscControlChars off, add WorkDirectory conf via rsyslog::params

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #145 from bogdando/rsyslog_tune

Rsyslog tune

bogdando committed:
Merge pull request #27 from Mirantis/master

Master sync

bogdando committed:
Merge pull request #28 from Mirantis/grizzly

Grizzly sync

bogdando committed:
Merge pull request #29 from Mirantis/centos-6.4

Centos 6.4 support for grizzly

bogdando committed:
Change 2.1->3.0

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #148 from bogdando/grizzly

Centos 6.4 support for grizzly

bogdando committed:
Edit README.rst for CI

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #149 from bogdando/grizzly

Edit README.rst for CI

bogdando committed:
Fix VIPs for non-HA cases

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Fix settings.py: change back to centos63.qcow2

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #33 from Mirantis/master

Master sync - kill deploy from manifests

bogdando committed:
Set none for default proxy

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #35 from monroid/iso_master

Added forwarding setting

bogdando committed:
Merge pull request #36 from bogdando/iso_master

Iso master

bogdando committed:
Merge pull request #38 from bogdando/iso_master

Set none for default proxy

bogdando committed:
Merge pull request #39 from Mirantis/grizzly

Grizzly sync back

bogdando committed:
Merge pull request #40 from Mirantis/centos-6.4

Centos 6.4

bogdando committed:
Merge pull request #41 from bogdando/fix_centos6.4

Fix centos6.4

bogdando committed:
Merge pull request #42 from bogdando/fix_vips_for_simple

Fix VIPs for non-HA cases

bogdando committed:
Edit Changelog

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #151 from bogdando/grizzly

Grizzly sync

bogdando committed:
Merge remote-tracking branch 'stable/master' into grizzly

Conflicts:
	fuel_test/cobbler/cobbler_test_case.py
	fuel_test/cobbler/test_compact.py
	fuel_test/cobbler/test_full.py
	fuel_test/cobbler/test_minimal.py
	fuel_test/cobbler/test_simple.py
	fuel_test/cobbler/test_swift.py
	fuel_test/manifest.py
	fuel_test/prepare.py
	fuel_test/settings.py

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Fix merge, edit CI README

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Set default parent_proxy to none

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #152 from bogdando/grizzly

Merged tests grizzly sync

bogdando committed:
CI add forwarding setting

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #153 from bogdando/grizzly

CI add forwarding setting

bogdando committed:
Fix typo

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge branch 'centos-6.4' of github.com:Mirantis/fuel into Mirantis-centos-6.4

Conflicts:
	deployment/puppet/openstack/manifests/mirantis_repos.pp

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #44 from bogdando/fix_centos6.4

Fix centos6.4

bogdando committed:
Merge pull request #154 from bogdando/grizzly

Grizzly fix centos 6.4

bogdando committed:
Change MIRRORs to Mirantis download

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #45 from bogdando/fix_centos6.4

Change MIRRORs to Mirantis download

bogdando committed:
Merge pull request #46 from Mirantis/centos-6.4

all repo to mirantis

bogdando committed:
Merge pull request #48 from bogdando/fix_centos6.4

Fix centos6.4

bogdando committed:
Merge pull request #156 from bogdando/grizzly

Grizzly sync iso/Makefile

bogdando committed:
Merge branch 'master' of github.com:Mirantis/fuel into Mirantis-master

Conflicts:
	fuel_test/settings.py

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge branch 'Mirantis-master' into grizzly

bogdando committed:
Fix merge for centos 6.4 grizzly

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #158 from bogdando/grizzly

Grizzly sync

bogdando committed:
Edit CI README

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #52 from Mirantis/master

Master sync

bogdando committed:
Add lost CLEAN

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #159 from bogdando/grizzly

Grizzly sync

bogdando committed:
Fix spacing

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #160 from bogdando/grizzly

Fix spacing

bogdando committed:
Merge pull request #55 from Mirantis/master

Master sync for tests

bogdando committed:
Merge remote-tracking branch 'stable/master' into grizzly

Conflicts:
	fuel_test/cobbler/vm_test_case.py
	fuel_test/settings.py

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #168 from bogdando/grizzly

Grizzly sync tests

bogdando committed:
Fix spacing

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Exclude puppet debug from compact

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #177 from bogdando/grizzly

Master testing merge request related fixes

bogdando committed:
Del yum-prio, del folsom repos, fix external repos

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Epel off, gpgcheck for external grizzly off

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Syslog and cinder (all) for test_simple

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
CI edit README

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
CI fix README

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Fix external repo for grizzly, disable cetnos-updates

Centos-Base from Mirantis includes Centos-Updates as merged one, no need for default updates repo.

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

Commit:
gitignore for idea

Commit:
Merge branch 'master' of github.com:Mirantis/fuel

Commit:
correct puppet function directory

vicglarson committed:
Merge pull request #192 from Zipfer/master

correct puppet func path

vicglarson committed:
Merge pull request #193 from xdel/master

puppet-3.0 cannot downgrade itself

bogdando committed:
Merge branch 'master' of github.com:Mirantis/fuel into Mirantis-master

Conflicts:
	fuel_test/settings.py

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge branch 'master' into grizzly

Conflicts:
	deployment/mcollective/astute/lib/astute/logparser/parser_patterns.rb
	deployment/mcollective/astute/lib/astute/logparser/provision.rb
	fuel_test/cobbler/test_full.py
	fuel_test/cobbler/test_single.py
	fuel_test/cobbler/test_swift.py
	fuel_test/settings.py

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Fix merge: DEBUG to PUPPET_AGENT_COMMAND

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #191 from bogdando/grizzly

Grizzly sync: fix repos, ci readme, master sync

bogdando committed:
Fix ubuntu mirror for iso/Makefile

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #194 from bogdando/grizzly

Fix ubuntu mirror for iso/Makefile

bogdando committed:
Del separate updates repo

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #57 from xdel/master

xdel sync for master, use puppet 2

bogdando committed:
Merge branch 'master' into grizzly

bogdando committed:
Fix settings.py amd README to puppet 2

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #196 from bogdando/grizzly

Grizzly sync, use puppet 2

bogdando committed:
Fix repos, get rid of osci

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Fix rsyslog params

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Merge pull request #197 from bogdando/grizzly

Grizzly sync

Commit:
DOC: Some changes in docs.

bogdando committed:
Change default keystone to PKI

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

bogdando committed:
Fix spacing

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

Commit:
Update prepare.py

bogdando committed:
Fix notify chains

Fixes: defect #FUEL-671

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

Commit:
added python-quantumclient

aglarendil committed:
Tests: change deploy timeout

aglarendil committed:
change default cinder_nodes default to empty array

aglarendil committed:
change test_repo to fuel-3.0

aglarendil committed:
fix cinder nodes setting

set manage_volumes to true if `$is_cinder_node` is set
remove all nova-volume related stuff
alter site*.pp files correspondingly

aglarendil committed:
fix swift-proxy for Grizzly

change authtoken egg
change signing dir for swift

Commit:
Corrected typos, changed .msk.mirantis.net to .example.com

Commit:
Merge pull request #202 from bogdando/fix671

Fix671

Commit:
Merge pull request #203 from aglarendil/grizzly

Fixes for grizzly support

Commit:
Merge branch 'grizzly' of github.com:Mirantis/fuel

aglarendil committed:
return nova-volume support for folsom versions

aglarendil committed:
fix possible python default value collection flaw

aglarendil committed:
make setup timeout configurable

Commit:
Crreate directory, that need for crm non-warning start.

Commit:
start Quantum ovs-agent on all controllers

Commit:
start DHCP and L3 agents on different controllers if it's possible

Commit:
rename resource

Commit:
If we create cloned resource -- we need use it.

xenolog committed:
fix git artefacts in documentation page.

Commit:
FIX:   add losted .to_i

Commit:
Merge branch 'grizzly__Q2' of github.com:xenolog/fuel into grizzly__Q2

aglarendil committed:
Revert "Change default keystone to PKI"

Look into OSCI-363

This reverts commit 7fd4c35dca3a50ad9cff02fd03644814005ae3ba.

Commit:
Merge pull request #212 from aglarendil/grizzly

Grizzly fixes

xdel committed:
switch to rabbitmq connector; from vicglarson/astute

vicglarson committed:
Merge pull request #214 from xdel/grizzly

switch to rabbitmq connector; from vicglarson/astute

Commit:
Merge branch 'grizzly' of github.com:Mirantis/fuel

bogdando committed:
Fix 880d620c commit

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

Commit:
Merge pull request #206 from xenolog/grizzly__Q2

start DHCP and L3 agents on different controllers if it's possible

vicglarson committed:
Set mode ha_full for all deployments. Update config.yaml in docs.

Conflicts:
	docs/pages/installation-instructions/config.yaml

Commit:
Merge branch 'grizzly' of github.com:Mirantis/fuel

vicglarson committed:
Delete deployment mode from common config

vicglarson committed:
Merge branch 'grizzly' of github.com:Mirantis/fuel into grizzly

vicglarson committed:
Sync config.yaml again

bogdando committed:
Fix README for tests, add pip-req for prepare.py

Signed-off-by: Bogdan Dobrelya <bogdando@mail.ru>

xdel committed:
Merge pull request #217 from vicglarson/grizzly_astute

One deployment mode in astute.yaml

Commit:
Merge branch 'grizzly' of github.com:Mirantis/fuel

Commit:
Clean up 'How installation works.'

Commit:
Minor changes to 'Before you start'.

Commit:
Added 3.0 release notes.

Commit:
Added Grizzly reference to download information.

Commit:
Fixed heading levels in reference architecture

Commit:
Refactored installation section for new structure.

Commit:
Removed Advanced Topics section

Commit:
Added new FAQ

Commit:
Corrections to install of VMs.

Commit:
Corrected typos

Commit:
Debugged installation procedure and checked for old IPs.

Commit:
Added orchestration note.

Commit:
Added manual deployment note.

Commit:
Fixed formatting issue.

Commit:
Typo correction
